RichString.c

```diff
@@ -73,10 +73,10 @@ static inline int RichString_writeFromAscii(RichString* this, int attrs, const c
    return len;
 }
 
-inline void RichString_setAttrn(RichString* this, int attrs, int start, int finish) {
+inline void RichString_setAttrn(RichString* this, int attrs, int start, int len) {
```

Comment:
len is a size entity, thus what about `size_t`?

Maybe also indicate if `len` specifies `bytes` or `cells`/`code points`/`characters`.

Reply:
renamed to `charcount`